server/cri/types/types.go

```diff
@@ -6,22 +6,28 @@ const (
 	RuntimeReady = "RuntimeReady"
 	NetworkReady = "NetworkReady"
 
-	NamespaceModePOD       NamespaceMode = 0
-	NamespaceModeCONTAINER NamespaceMode = 1
-	NamespaceModeNODE      NamespaceMode = 2
-	NamespaceModeTARGET    NamespaceMode = 3
-
-	MountPropagationPropagationPrivate         MountPropagation = 0
-	MountPropagationPropagationHostToContainer MountPropagation = 1
-	MountPropagationPropagationBidirectional   MountPropagation = 2
-
-	ContainerStateContainerCreated ContainerState = 0
-	ContainerStateContainerRunning ContainerState = 1
-	ContainerStateContainerExited  ContainerState = 2
-	ContainerStateContainerUnknown ContainerState = 3
-
-	PodSandboxStateSandboxReady    PodSandboxState = 0
-	PodSandboxStateSandboxNotReady PodSandboxState = 1
+	NamespaceModePOD NamespaceMode = iota
```

Comment:
this PR is a good idea, but I think we should drop the types changes in favor of https://github.com/cri-o/cri-o/pull/5471